### ISSUE TITLE
Missing keepHeldItem option for Deploying recipes

### ISSUE THREAD

**Issue by PssbleTrngle** [KubeJS-Mods/KubeJS-Create#14]
Deploying recipes now include a boolean property named `keepHeldItem` which decides wether the held item will get used or remains.
There should be a way to set this property, for example
```javascript
event.recipes
   .createDeploying(waxed, [unwaxed,"minecraft:honeycomb_block"])
   .keepHeldItem();
```

**Comment by MaxNeedsSnacks** [comment]
@Prunoideae was working on an update to KJS Create as far as I know, pretty sure they included that in there as well

**Comment by MaxNeedsSnacks** [comment]
Implemented in #17 ^^
